app/src/app/api/v0/inventory/[inventory]/download/route.ts

```diff
@@ -129,6 +133,16 @@ export const GET = apiHandler(async (req, { params, session }) => {
         "Content-Disposition": `attachment; filename="inventory-${inventory.city.locode}-${inventory.year}.xls"`,
       };
       break;
+    case "ecrf":
+      body = await ECRFDownloadService.downloadECRF(
+        output as InventoryWithInventoryValuesAndActivityValues,
+        lng,
+      );
+      headers = {
+        "Content-Type": "application/vnd.ms-excel",
+        "Content-Disposition": `attachment; filename="eCRF_inventory-${inventory.city.locode}-${inventory.year}.xlsx"`,
```

Comment:
Let's use - and _ consistently in the file name.
```suggestion
        "Content-Disposition": `attachment; filename="eCRF-inventory-${inventory.city.locode}-${inventory.year}.xlsx"`,
```